metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
```

Comment:
Maybe `sb_cape_cin` or `sfcbased_cape_cin`? Not sure what the best spelling is here, but I like the more descriptive of the two personally.